test/persistence-hooks.suite.js

```diff
@@ -575,6 +576,204 @@ module.exports = function(dataSource, should) {
           done();
         });
       });
+
+      describe('persist hook', function() {
```

Comment:
The test file structures tests by DAO method (create, update), not by hooks. Please preserve that structure and move the tests from `describe('persist hook')` to appropriate per-method suites.